samples/ControlCatalog/Views/NotificationView.xaml

```diff
@@ -0,0 +1,19 @@
+<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             x:Class="ControlCatalog.Views.NotificationView">
+    <Border Padding="12" MinHeight="20" Background="DodgerBlue">
```

Comment:
So the notfication needs to be defined in the application - there is no default theme for notifications?

Reply:
Iv renamed the xaml files to make it more obvious that there is a default theme, see notificationcard.xaml... this is a user defining their own content for managed notifications.